pkgs/data/fonts/termsyn/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, mkfontscale }:
+
+stdenv.mkDerivation rec {
+  pname = "termsyn";
+  version = "1.8.7";
+
+  src = fetchurl {
+    url = "https://pilotfiber.dl.sourceforge.net/project/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "15vsmc3nmzl0pkgdpr2993da7p38fiw2rvcg01pwldzmpqrmkpn6";
+  };
+
+  nativeBuildInputs = [ mkfontscale ];
+
+  installPhase = ''
+    install -m 644 -D *.pcf -t "$out/share/fonts/local"
+    install -m 644 -D *.psfu -t "$out/share/kbd/consolefonts"
+    mkfontdir "$out/share/fonts/local"
```

Comment:
Why local?

Reply:
I installed to local because the build instructions in the readme say to install the .pcf files to /usr/share/fonts/local; is there a better location to keep them in?

Reply:
The instructions need to be adapted a bit for nixpkgs. We do not have the concept of ``/usr/local/``, or ``/usr/share/local`` because we don't need to. Our package manager is smart enough to cleanup after itself and you cannot manually copy things into those directories. So the local can be removed to keep things clean.

Reply:
Ah I understand now, thanks. I've updated the commit to do so.